more/free-programming-interactive-tutorials-en.md

```diff
@@ -379,6 +380,11 @@
 * [Rustlings](https://github.com/rust-lang/rustlings)
 
 
+### Salesforce
+
+* [Salesforce Solutioning] (https://www.pluralsight.com/courses/play-by-play-diagramming-salesforce-solutions by Matthew Morris and Don Robins)
```

Comment:
I have a suggestion, wherever you put this course, update it like this, else your provided Link will fail to work.
```suggestion
* [Salesforce Solutioning] (https://www.pluralsight.com/courses/play-by-play-diagramming-salesforce-solutions) - Matthew Morris and Don Robins
```